webpack/ForemanWebhooks/Routes/Webhooks/Components/WebhookForm/Components/ForemanFormikField.js

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Field as FormikField } from 'formik';
+
+import FormField from 'foremanReact/components/common/forms/FormField';
+
+const ForemanFormikField = ({
+  name,
+  type,
+  required,
+  label,
+  labelHelp,
+  inputSizeClass,
+  rows,
+  placeholder,
+  options,
+}) => (
+  <FormikField name={name}>
+    {({
+      field, // { name, value, onChange, onBlur }
+      form: { touched, errors }, // also values, setXXXX, handleXXXX, dirty, isValid, status, etc.
+    }) => {
+      const content =
+        type === 'textarea' ? (
+          <textarea
+            {...field}
+            className="form-control"
+            rows={rows}
+            placeholder={placeholder}
+          />
+        ) : (
+          <input
+            {...field}
+            type={type}
+            checked={type === 'checkbox' ? field.value || '' : undefined}
+            className={type === 'checkbox' ? '' : 'form-control'}
+          />
+        );
+      return (
+        <FormField
+          {...field}
+          error={errors[name]}
+          type={type}
+          inputSizeClass={inputSizeClass}
+          required={required}
+          label={label}
+          labelHelp={labelHelp}
+          options={options}
+        >
+          {type === 'select' ? null : content}
+        </FormField>
+      );
+    }}
+  </FormikField>
+);
+
+ForemanFormikField.propTypes = {
+  name: PropTypes.string.isRequired,
+  type: PropTypes.string.isRequired,
+  required: PropTypes.bool,
+  label: PropTypes.string.isRequired,
+  labelHelp: PropTypes.string,
+  inputSizeClass: PropTypes.string,
+  rows: PropTypes.number,
+  placeholder: PropTypes.string,
+  options: PropTypes.array,
+};
+
+ForemanFormikField.defaultProps = {
+  required: false,
+  labelHelp: null,
+  inputSizeClass: 'col-md-4',
```

Comment:
As the form is inside a small modal maybe this should be changed to 8 or 10?
I used a short url and it wasn't fully visible 
![Screenshot from 2021-02-22 18-04-40](https://user-images.githubusercontent.com/30431079/108734760-723ab300-7538-11eb-9cb8-3dfe20f0f384.png)

Reply:
Totally makes sense, changed to `8`.